protonvpn_cli/utils.py

```diff
@@ -15,10 +15,8 @@
 # ProtonVPN-CLI functions
```

Comment:
build.build.sh runs a fmt, yapf, and SA, mypy. All fmt changes where done by yapf. Extra comments were added by me as exceptions to mypy.